cmd/eventlistenersink/main.go

```diff
@@ -18,23 +18,56 @@ package main
 
 import (
 	"fmt"
+	"go.uber.org/zap"
 	"log"
 	"net/http"
 
+	"github.com/tektoncd/triggers/pkg/logging"
 	"github.com/tektoncd/triggers/pkg/sink"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+	"knative.dev/pkg/signals"
+)
+
+// EventListenerLogKey is the name of the logger for the eventlistener cmd
+const (
+	EventListenerLogKey = "eventlistener"
```

Comment:
Oops, I think I missed this before, but the comment should be directly above the `EventListenerLogKey` line.
```
// EventListenerLogKey is the name of the logger for the eventlistener cmd
EventListenerLogKey = "eventlistener"
```

Reply:
oh, I fix it.